db/entity/DiscordServerConfig.ts

```diff
@@ -1,12 +1,21 @@
-import { Entity, Column, PrimaryColumn, OneToMany } from "typeorm";
+import {
+  Entity,
+  Column,
+  PrimaryColumn,
+  PrimaryGeneratedColumn,
+  OneToMany,
+} from "typeorm";
 import type { StarkyModuleConfig } from "../../starkyModules/types";
 import { DiscordMember } from "./DiscordMember";
 
 @Entity()
-export class DiscordServer {
-  @PrimaryColumn()
+export class DiscordServerConfig {
```

Comment:
maybe we should add some kind of constraint : only one config per server and per role? let's discuss